src/config/dynamic-theme-fixes.config

```diff
@@ -403,6 +403,15 @@ INVERT
 
 ================================
 
+marketplace.visualstudio.com
+
+CSS
+.details-tab {
+  color: rgba(201, 199, 197, 0.9);
```

Comment:
Please, check how this fix looks in Light mode too. I've added a pattern for applying settings values to color, it looks like `${rgba(201, 199, 197, 0.9)}`, where the color inside `${}` is a source color without modifications. Also `!important` keyword should be used. So, your fix will look like `color: ${rgba(201, 199, 197, 0.9)} !important;`.